docs/general/information-schema.rst

```diff
@@ -124,7 +124,7 @@ number of replicas.
     | sys                | summits                 | BASE TABLE |             NULL | NULL               |
     | sys                | users                   | BASE TABLE |             NULL | NULL               |
     +--------------------+-------------------------+------------+------------------+--------------------+
-    SELECT 56 rows in set (... sec)
+    SELECT 57 rows in set (... sec)
```

Comment:
curious: how is this related to this PR?

Reply:
not related, but I decided to fix it anyway - we have this failure in the main LR [branch](https://github.com/crate/crate/pull/11832/commits) -